src/theory/quantifiers/ce_guided_pbe.cpp

```diff
@@ -202,16 +208,15 @@ void CegConjecturePbe::initialize( Node n, std::vector< Node >& candidates, std:
   if( !d_is_pbe ){
     Trace("sygus-unif") << "Do not do PBE optimizations, register..." << std::endl;
     for( unsigned i=0; i<candidates.size(); i++ ){
-      d_qe->getTermDatabaseSygus()->registerMeasuredTerm(candidates[i],
-                                                         d_parent);
+      d_qe->getTermDatabaseSygus()->registerEnumerator(
+          candidates[i], candidates[i], d_parent);
```

Comment:
Just to be sure, `candidates[i], candidates[i]` is intended here?

Reply:
Yes. This is the case where a function to synthesize has a single enumerator.